AUTHORS

```diff
@@ -0,0 +1,304 @@
+A S Alam <apreet.alam@gmail.com>
```

Comment:
This file has tons of duplicate names, and why do we need this?

Reply:
This I did not understand, "duplicate" I thought this file is getting generated automatically noh?

Reply:
This file is not in master. I think a tool that you're using is generating this file, and it's done so quite recently because there's some newer names in this file. I don't know why we'd need this file. By duplicate names, I mean there's places where the same name shows up multiple times with different email addresses, like gaetan semet with a multitude of different email addresses.